src/other/install/abis/known-005/erc_01594.json

```diff
@@ -143,7 +143,7 @@
       },
       {
         "name": "ret_1",
-        "type": "byte"
+        "type": "bytes1"
```

Comment:
Very strange. How did you find this? What is this? Is a byte and a bytes1 different things?

Reply:
The parser failed. These are the same thing, but `byte` has been removed in Solidity 8 in favour of `bytes1`: https://docs.soliditylang.org/en/latest/080-breaking-changes.html#silent-changes-of-the-semantics (last bullet point)